src/node_contextify.cc

```diff
@@ -69,6 +71,11 @@ using v8::WeakCallbackInfo;
 
 namespace {
 
+inline const char* ToString(Local<Value> value) {
+  String::Utf8Value utf8(value);
+  return *utf8;
+}
```

Comment:
Uh – This won’t work. :/ Once `utf8` goes out of scope, it also potentially frees the memory used for storing `value` in its UTF-8 form … I think you need to inline this everywhere (i.e. create distinct `Utf8Value`s for the different values).

Reply:
Or, alternatively, you could probably do this (warning, haven’t compiled or tried it):

```
std::ostream& operator<<(std::ostream& os, Local<Value> value) {
  String::Utf8Value utf8(value);
  return os << *utf8;
}
```

Reply:
Thanks. That seems to work.